src/ui/flutter_app/lib/services/mill/src/recorder.dart

```diff
@@ -21,7 +21,18 @@ part of '../mill.dart';
 class _GameRecorder extends PointedList<ExtMove> {
   _GameRecorder() : super();
 
-  // TODO: [Leptopoda] as this is not related to the recorder in general we should probably move this to the import export service.
+  @override
+  String toString() {
+    final buffer = StringBuffer("[ ");
```

Comment:
The `toString` method is nothing we'll use for the game. The toString of a list is commonly represented with square brackets and as this is a list (PointedList) we will print it as such. 

The method is only used when you want to print out the contents of the list and we either use this in logs (not now) but I use it to verify the test result of the import as we otherwise have no way to verify that the list has been popilated.